src/matrixrtc/RTCEncryptionManager.ts

```diff
@@ -114,6 +129,7 @@ export class RTCEncryptionManager implements IEncryptionManager {
     public leave(): void {
         this.transport.off(KeyTransportEvents.ReceivedKeys, this.onNewKeyReceived);
         this.transport.stop();
+        this.participantKeyRings.clear();
```

Comment:
The legacy encryption manager was only clearing its own key here, but I don't see why?